plugins/deezloader.py

```diff
@@ -118,6 +97,20 @@ async def deezload(message: Message):
         except Exception as e_r:
             await message.edit("Song not Found 🚫")
             await Clogger.log(f"#ERROR\n\n{e_r}")
+        await message.delete()
+        shutil.rmtree(TEMP_PATH, ignore_errors=True)
+        return
+
+    try:
+        if 'track/' in input_link:
+            await proper_trackdl(input_link, quality, message, loader, TEMP_PATH)
+        elif 'album/' or 'playlist/' in input_link:
```

Comment:
this works ?

Reply:
need brackets

Reply:
`it works`: yes

`is it the right way`: yeah No. 😂